plugin/storage/factory.go

```diff
@@ -249,3 +252,11 @@ func (f *Factory) Close() error {
 	}
 	return multierror.Wrap(errs)
 }
+
+func (f *Factory) setExpvarOptions() {
```

Comment:
nit: there's nothing specific to expvar in this function and others like it, consider a different name, like publishParams